File: CppPractice/CppLeetcode/Leetcode1945.cpp
#include<vector>
#include<string>
using namespace std;
struct ListNode
{
    int val;
    ListNode *next;
    ListNode():val(0),next(nullptr){};
    ListNode(int x):val(x),next(nullptr){};
    ListNode(int x, ListNode *p):val(x),next(p){};

};

class Solution {
public:
    int getLucky(string s, int k) 
    {
        string convertedString = "";
        for(char c:s)
        {
            convertedString = convertedString + stoi(c);
        }

        int theSum = 0;
        while (k>0)
        {
            for (char c:convertedString)
            {
                theSum += stoi(c);
            }
            convertedString = to_string(theSum);
            
            k--;
        }
        return stoi(convertedString);
    }
};

File: CSharpPractice/LeetcodePractice/Leetcode1945.cs
public class Solution1945 {
    public int GetLucky(string s, int k) 
    {
        string covertedString = "";
        foreach (char c in s)
        {
            covertedString += (c-'a'+1).ToString();
        }
        int theSum = 0;
        while (k>0)
        {
            foreach (char c in covertedString)
            {
                theSum += c-'a'+1;
            }
            covertedString = theSum.ToString();

            k--;
            
        }
        return int.Parse(covertedString);
        
    }
}